docs/cmdline-opts/location-trusted.md

```diff
@@ -2,7 +2,7 @@
 c: Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
 SPDX-License-Identifier: curl
 Long: location-trusted
-Help: As --location, but send auth to other hosts
+Help: As --location, but send auth and cookie to other hosts
```

Comment:
In the name of trying to keep this is brief as possible (for a better `curl -h` experience):

```suggestion
Help: As --location, but send secrets to other hosts
```

Also of course because *normally* curl will send cookies to the other host as well. It is only if you set cookies with `-H Cookie:` or with `-b name=moo` this exception goes for.

Reply:
It looks like this only effects on `-H Cookie:` but not `-b`.

Reply:
Yes, but my suggestion is the same anyway.